api/swagger-spec/v1.json

```diff
@@ -13188,18 +13188,11 @@
    },
    "v1.PodSecurityContext": {
     "id": "v1.PodSecurityContext",
-    "description": "PodSecurityContext holds pod-level security attributes and common container settings.",
+    "description": "PodSecurityContext holds pod-level security attributes and common container settings. Some fields are also present in SecurityContext.  Field values of SecurityContext take precedence over field values of PodSecurityContext.",
```

Comment:
Should you say "Some fields are also present in container level SecurityContext" it might not be clear to people reading this that SecurityContext really means ContainerSecurityContext